src/main/java/org/reactome/release/qa/common/QACheckerHelper.java

```diff
@@ -273,192 +278,43 @@ public static String getAttributeTableName(String clsName,
+		reactionPEs.addAll(reaction.getAttributeValuesList(ReactomeJavaConstants.catalystActivity));
+		reactionPEs.addAll(reaction.getAttributeValuesList(ReactomeJavaConstants.regulatedBy));
 
-	/**
-	 * Finds all PhysicalEntities that exist within an EntitySet by searching for all member PEs (via 'hasMember' attribute)
-	 * and all candidate PEs (if entitySet is a CandidateSet; via 'hasCandidate attribute).
-	 * @param entitySet GKInstance -- EntitySet instance that will be searched for all PhysicalEntities.
-	 * @return Set<GKInstance> -- All distinct PhysicalEntities that are found in EntitySet.
-	 * @throws Exception -- Thrown by MySQLAdaptor.
-	 */
-	public static Set<GKInstance> findEntitySetPhysicalEntities(GKInstance entitySet) throws Exception {
-		Set<GKInstance> physicalEntities = new HashSet<>();
-		// All EntitySet instances have a 'hasMember' slot to be searched.
-		Set<String> entitySetAttributes = new HashSet<>(Arrays.asList(ReactomeJavaConstants.hasMember));
-		// If the EntitySet is a CandidateSet, 'hasCandidate' attribute needs to be searched as well.
-		if (entitySet.getSchemClass().isa(ReactomeJavaConstants.CandidateSet)) {
-			entitySetAttributes.add(ReactomeJavaConstants.hasCandidate);
+		Set<GKInstance> allReactionPEs = new HashSet<>();
+		for (GKInstance reactionPE : reactionPEs) {
+			allReactionPEs.add(reactionPE);
+			allReactionPEs.addAll(getPhysicalEntityContainedInstances(reactionPE));
 		}
-		for (String entitySetAttribute : entitySetAttributes) {
-			// Recursively searches members/candidates in EntitySet for all PhysicalEntities.
-			for (GKInstance setInstance : (Collection<GKInstance>) entitySet.getAttributeValuesList(entitySetAttribute)) {
-				physicalEntities.addAll(findAllPhysicalEntities((setInstance)));
-			}
-		}
-		return physicalEntities;
+		return allReactionPEs;
 	}
 
 	/**
-	 * Checks if incoming Event is manually inferred by checking inferredFrom referral.
-	 * @param event GKInstance -- Event instance being checked for inferredFrom referral.
-	 * @return boolean -- true if inferredFrom referral exists, false if not.
+	 * This method is wrapper for the InstanceUtilities getContainedInstances method.
+	 * @param reactionPE GKInstance -- PhysicalEntity that will be checked for contained instances.
+	 * @return Set<GKInstance> -- All distinct contained instances in the incoming PhysicalEntity.
 	 * @throws Exception -- Thrown by MySQLAdaptor.
 	 */
-	public static boolean manuallyInferred(GKInstance event) throws Exception {
-		return event.getReferers(ReactomeJavaConstants.inferredFrom) != null;
+	public static Set<GKInstance> getPhysicalEntityContainedInstances(GKInstance reactionPE) throws Exception {
+		return InstanceUtilities.getContainedInstances(
+				reactionPE,
+				ReactomeJavaConstants.hasMember,
+				ReactomeJavaConstants.hasCandidate,
+				ReactomeJavaConstants.hasComponent,
+				ReactomeJavaConstants.repeatedUnit,
+				ReactomeJavaConstants.activeUnit);
```

Comment:
Nice - I like that this is one line per attribute!  One nit-picky style thing you could consider is putting the closing parenthesis on its own line with the same indentation as line 311.